From 4faf26a3d5fe940044ce7d05b25d017c6e1f5670 Mon Sep 17 00:00:00 2001
From: Fabio Porcedda <fabio.porcedda@gmail.com>
Date: Sun, 22 Feb 2015 09:03:38 +0100
Subject: [PATCH] Move -shared flag from $ldflags to $lddflags

This is useful because when a flag is copied from $ldflags to $lddflags
the default -shared flag is gone, so permit to add it back.
---
 cnf/configure_misc.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cnf/configure_misc.sh b/cnf/configure_misc.sh
index 6386152..b98fe90 100644
--- a/cnf/configure_misc.sh
+++ b/cnf/configure_misc.sh
@@ -8,7 +8,7 @@ if [ -n "$ldflags" -a "$x_lddlflags" != "user" ]; then
 	msg "Checking which flags from \$ldflags to move to \$lddlflags"
 	for f in $ldflags; do 
 		case "$f" in
-			-L*|-R*|-Wl,-R*)
+			-L*|-R*|-Wl,-R*|-shared)
 				msg "\tadded $f"
 				appendvar 'lddlflags' "$f"
 				;;
-- 
2.3.0

